include/lug/Window/Window.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <memory>
+#include <queue>
+#include <lug/Window/Event.hpp>
```

Comment:
Include `WindowHandle.hpp` because you will use it and that will check if its content compiles.